.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Please add this file to the .gitignore file. This is a local config file that does not need to be checked into the repository.